palisade-service/src/test/java/uk/gov/gchq/palisade/service/palisade/service/AuditServiceTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.palisade.service.palisade.service;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.slf4j.LoggerFactory;
+
+import uk.gov.gchq.palisade.service.palisade.request.AuditRequest;
+import uk.gov.gchq.palisade.service.palisade.web.AuditClient;
+
+import java.util.List;
+import java.util.concurrent.Executor;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AuditServiceTest {
+    private Logger logger;
+    private ListAppender<ILoggingEvent> appender;
+    private AuditService auditService;
+
+    @Mock
+    AuditClient auditClient;
+    @Mock
+    Executor executor;
+
+    @Before
+    public void setUp() {
+        logger = (Logger) LoggerFactory.getLogger(AuditService.class);

```

Comment:
Unecessary cast

Reply:
The Logback Loggers allow us to add appenders and listen to logging statements, but not in slf4j

This is more verbosely ch.qos.logback.Logger logger = org.slf4j.LoggerFactory.getLogger(~.class);

Reply:
The name is a bit misleading, but the cast is needed